lib/cc/engine/analyzers/violation.rb

```diff
@@ -19,7 +19,8 @@ def format
             "location": format_location,
             "remediation_points": calculate_points,
             "other_locations": format_other_locations,
-            "content": content_body
+            "content": content_body,
+            "fingerprint": "#{current_sexp.file}-#{current_sexp.mass}"
```

Comment:
Fingerprints are generally hashes. What do you think of having a `fingerprint` method that hashes this string?